addons/l10n_id_efaktur/views/account_move_views.xml

```diff
@@ -6,16 +6,14 @@
             <field name="model">account.move</field>
             <field name="inherit_id" ref="account.view_move_form"/>
             <field name="arch" type="xml">
-                <field name="partner_shipping_id" position="before">
-                    <field name="l10n_id_need_kode_transaksi" invisible="1"/>
-                    <field name="l10n_id_kode_transaksi" invisible="country_code != 'ID' or not l10n_id_need_kode_transaksi" readonly="state in ['cancel', 'posted']" required="l10n_id_need_kode_transaksi"/>
-                    <field name="l10n_id_efaktur_range" invisible="country_code != 'ID' or state != 'draft' or l10n_id_available_range_count &lt; 2 or not l10n_id_need_kode_transaksi" options="{'no_create': True}"/>
-                </field>
                 <button name="button_draft" position="after">
                     <button name="reset_efaktur" string="Reset E-Faktur" type="object" invisible="country_code != 'ID' or not l10n_id_tax_number or state != 'cancel' or l10n_id_efaktur_document"/>
                 </button>
                 <xpath expr=".//group[@id='other_tab_group']" position="inside">
                     <group string="Electronic Tax" invisible="country_code != 'ID'">
+                        <field name="l10n_id_need_kode_transaksi" invisible="1"/>
```

Comment:
I thought we didn't need invisible fields for the modifiers anymore? :thinking: 
You don't have `l10n_id_show_kode_transaksi`. Are you keeping it just for the stable version? (it avoids crashed but it sill moves it, so then I would keep that fields where it is)